app/Database/Tables.hs

```diff
@@ -230,6 +243,13 @@ instance ToJSON Meeting where
       drop 7
   }
 
+instance ToJSON Times where
+  toJSON = genericToJSON defaultOptions {
+    fieldLabelModifier =
```

Comment:
I'm not sure why this part is necessary. Can't we just use the original field options?